frontend/src/views/FinalSupplyEquipments/_schema.jsx

```diff
@@ -60,6 +60,7 @@ export const finalSupplyEquipmentColDefs = (
       minDate: moment(`${compliancePeriod}-01-01`, 'YYYY-MM-DD').toDate(),
       maxDate: moment(`${compliancePeriod}-12-31`, 'YYYY-MM-DD').toDate()
     },
+    cellEditorPopup: true,
```

Comment:
Useful for components like date pickers, which need additional space to display without overlapping or being constrained by the cell boundaries